crypto/multisig/multisignature.go

```diff
@@ -4,20 +4,21 @@ import (
 	"errors"
 
 	"github.com/tendermint/tendermint/crypto"
+	bA "github.com/tendermint/tendermint/crypto/multisig/bitarray"
```

Comment:
This alias does more harm than good, obscuring what is imported here.

Reply:
Ok!